hedera-node/src/main/java/com/hedera/services/grpc/marshalling/RoyaltyFeeAssessor.java

```diff
@@ -74,7 +76,8 @@ public ResponseCodeEnum assessAllRoyalties(
                     // A NFT transfer with royalty fees to an unknown alias is not possible, since
                     // the auto-created
                     // account will not have any hbar to pay the fallback fee
-                    if (change.hasNonEmptyCounterPartyAlias()) {
+                    if (dynamicProperties.areTokenAutoCreationsEnabled()
```

Comment:
Here also I think we can skip this check, and only do it once during `handleTransaction` in `AutoCreationLogic` as discussed.

Reply:
yeah agreed, changed to `AutoCreationLogic`. Thanks @tinker-michaelj